client/src/Components/EditProfile.js

```diff
@@ -1,10 +1,133 @@
+    },
+    avatar: {
+      margin: theme.spacing(1),
+      backgroundColor: theme.palette.secondary.main,
+    },
+    form: {
+      width: '100%', 
+      marginTop: theme.spacing(3),
+    },
+    submit: {
+      margin: theme.spacing(3, 0, 2),
+    },
+  }));
+  const classes = useStyles()
+
+  return (
+   
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Typography component="h1" variant="h5">
+          Edit profile
+        </Typography>
+        <form className={classes.form} noValidate>
+          <Grid container spacing={2}>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                name="firstName"
+                variant="outlined"
+                required
+                fullWidth
+                id="firstName"
+                label="First Name"
+                value={this.props.value}
+                onChange={this.props._change}
+                
+              />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                id="lastName"
+                label="Last Name"
+                name="lastName"
+                value={this.props.value}
+                onChange={this.props._change}
+
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                id="email"
+                label="Email Address"
+                name="email"
+                value={this.props.value}
+                onChange={this.props._change}
+
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                name="password"
+                label="Password"
+                type="password"
+                id="password"
+                value={this.props.value}
+                onChange={this.props._change}
+
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                variant="outlined"
+                required
+                fullWidth
+                name="phonenumber"
+                label="Phone Number"
+                type="phonenumber"
+                id="phonenumber"
+                value={this.props.value}
+                onChange={this.props._change}
+
+              />
+            </Grid>
+            
+          </Grid>
+          <Button
+            type="submit"
+            fullWidth
+            variant="contained"
+            color="primary"
+            className={classes.submit}
+          >
+            Confirm Changes
+          </Button>
+        </form>
+      </div>
+      
+    </Container>
+   
+  );
+};
+export default EditProfileUser;
```

Comment:
You defined the function as just `EditProfile` so this is currently exporting something that doesn't exist and will crash.